### PR TITLE
Make JSON and Markdown formatting consistent across all samples

### PR DESCRIPTION
**Pull request by thomasturrell**
* Fix some minor JSON syntax errors

### CODE REVIEW

**Comment by thomasturrell** [comment]
When reviewing this pull request it might be useful to ignore whitespace changes so you can see the syntax fixes.

I have used [Prettier](https://prettier.io) which is a Visual Studio Code plug in to format the JSON.

File: samples/scenarios/18-cmi5-allowed-passed/03-cmi5LearnerPreferences_data.json [hunk 1]
```diff
@@ -1,4 +1,4 @@
 {
-  "languagePreference": "en-US", "en-GB",
+  "languagePreference": "en-US en-GB",
```
Comment (brianjmiller) [inline samples/scenarios/18-cmi5-allowed-passed/03-cmi5LearnerPreferences_data.json]: This is invalid. (not that it was correct before)

**Comment by brianjmiller** [comment]
If we're going to make this significant of a change I'd also advocate for dropping the properties in the statements that are from them being captured in the LRS as opposed to what is to be sent. Things like `stored`, and `authority`. The original samples didn't include those since it was intended to be what would be sent from the AU/LMS rather than what would be read back out afterwards. Though they should have an understanding of xAPI I'd rather people not think they can set those properties based on these samples.

File: samples/scenarios/19-cmi5-allowed-test-questions/03-cmi5LearnerPreferences_data.json [hunk 2]
```diff
@@ -1,4 +1,4 @@
 {
-  "languagePreference": "en-US", "en-GB",
+  "languagePreference": "en-US en-GB",
```
Comment (brianjmiller) [inline samples/scenarios/19-cmi5-allowed-test-questions/03-cmi5LearnerPreferences_data.json]: +1 invalid.

File: samples/scenarios/19-cmi5-allowed-test-questions/index.md [hunk 3]
```diff
@@ -52,41 +52,38 @@
 {% include_relative 04-initialized_statement.json %}
 </pre>
 
-
-
-## answered allowed statement  1 of 4)
+## answered allowed statement 1 of 4)
```
Comment (brianjmiller) [inline samples/scenarios/19-cmi5-allowed-test-questions/index.md]: Are these supposed to read `(1 of 4)`? Note opening paren.

**Comment by thomasturrell** [comment]
> I'd also advocate for dropping the properties in the statements that are from them being captured in the LRS as opposed to what is to be sent. Things like `stored`, and `authority`.

I completely agree. I'm happy to do it, let's discuss on working group call.

**Comment by cawerkenthin** [comment]
👍

**Comment by MrBillMcDonald** [comment]
Closed per Aug 13th Meeting

**Pull request closed without merging**